docs/how-to-guides/admin/privacycookie.rst

```diff
@@ -51,3 +51,29 @@ All the adjustments should be provided as values in ``ui.privacy.banner`` as fol
 You can refer to the current
 `values.yaml in the renku-ui repository <https://github.com/SwissDataScienceCenter/renku-ui/blob/master/helm-chart/renku-ui/values.yaml>`_
 to check the default values and to find an example of how to apply different settings.
+
+Dashboard Message
+~~~~~~~~~~~~~~~~~
+
+The UI can display a configurable message for logged in users on their dashboard
+page, suited for showing general information about Renku.
+
+This feature can be enabled and configured by editing the values found in
+``ui.client.dashboardMessage``. Set ``ui.client.dashboardMessage.enabled``
+to ``true`` to enable have the message displayed on the dashboard page.
```

Comment:
I would drop "have" or change to "to enable _having_ the message displayed".

Or at least Grammarly tells me so :joy: